features/number.yml

```diff
@@ -0,0 +1,85 @@
+name: Number
+description: The number type (and `Number` object) represents floating-point numbers, such as 42 or -4.201. JavaScript numbers can also represent boundless negative and positive values as `Infinity` or not-a-number as `NaN` (as in `0 * Infinity`).
+spec: https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-number-objects
+group: primitive-types
+status:
+  compute_from: javascript.builtins.Number
+compat_features:
+  - javascript.builtins.Infinity
+  - javascript.builtins.isNaN
+  - javascript.builtins.Math
+  - javascript.builtins.Math.abs
+  - javascript.builtins.Math.acos
+  - javascript.builtins.Math.acosh
+  - javascript.builtins.Math.asin
+  - javascript.builtins.Math.asinh
+  - javascript.builtins.Math.atan
+  - javascript.builtins.Math.atan2
+  - javascript.builtins.Math.atanh
+  - javascript.builtins.Math.cbrt
+  - javascript.builtins.Math.ceil
+  - javascript.builtins.Math.clz32
+  - javascript.builtins.Math.cos
+  - javascript.builtins.Math.cosh
+  - javascript.builtins.Math.E
+  - javascript.builtins.Math.exp
+  - javascript.builtins.Math.expm1
+  - javascript.builtins.Math.f16round
```

Comment:
Let's drop this one too. I'll link to this PR from the Float16Array PR.

Reply:
done